README.md

```diff
@@ -1 +1,59 @@
-# Week-One-Capstone
\ No newline at end of file
+# ![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Capstone
+![screenshot](Capture.JPG)
+
+
```

Comment:
- [ ] Kindly make sure you have a detailed description telling what the project is about. as it will help document your work in a professional way.

Reply:
Corrected!